hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperPaths.java

```diff
@@ -35,6 +36,10 @@ public String basePath() {
         return basePath;
     }
 
+    public String extractChildNode(String fullChildPath, String basePath) {
+        return fullChildPath.substring(basePath.length() + URL_SEPARATOR.length());
```

Comment:
you have `basePath` set in `this` already, so the argument can be omitted.

Reply:
Argument `basePath` is not the same as `this.basePath`. I will rename this argument.

Reply:
ok :)

Reply:
Done